### ISSUE TITLE
Review checkboxes and radio attr/prop usage

### ISSUE THREAD

**Issue by tadatuta** [bem/bem-components#1886]
It turned out that `form.reset()` works wrong with [current way](https://github.com/bem/bem-components/blob/v3/common.blocks/checkbox/checkbox.js#L17-L19) of checking/unchecking checkboxes.

Such variant was introduced in https://github.com/bem/bem-components/commit/012de48146234dc5c1ea47297687046311a0ac6b as a fix for previous change made for a11y: https://github.com/bem/bem-components/commit/db4a1385029ee328550aaacb541b41e021381fea

Is there any way to satisfy a11y and get rid of `attr()` setter?

**Comment by tadatuta** [comment]
cc @aristov

**Comment by narqo** [comment]
According to [WAI-ARIA Authoring Practices for Checkbox](https://www.w3.org/TR/wai-aria-practices-1.1/#checkbox) widget, it should be possible to use `aria-checked` attribute.
